chart/cohort/values.yaml

```diff
@@ -31,4 +31,8 @@ service:
     - 9080
     - 9443
 
-internalTlsCertSecretName: cohort-services-tls-secret
\ No newline at end of file
+internalTlsCertSecretName: cohort-services-tls-secret
+# enableDarkFeatures is used to enable or disable REST endpoints depending on the env. we are deploying to
+# there are some REST endpoints we only want enabled in a development environment. The value ends up
+# in the jvm.options file the liberty server
+enableDarkFeatures: REPLACEDBYOVERRIDE
```

Comment:
Would there be any value to setting this feature to `false`/`all disabled` by default in  the chart? Then dark features would have to  be turned on  explicitly in an override.

Reply:
I would be fine doing that. Good suggestion.